flink-scala/src/main/scala/org/apache/flink/api/scala/typeutils/OptionSerializer.scala

```diff
@@ -65,7 +65,7 @@ class OptionSerializer[A](val elemSerializer: TypeSerializer[A])
     case Some(a) =>
       target.writeBoolean(true)
       elemSerializer.serialize(a, target)
-    case None =>
+    case _ =>
```

Comment:
Yes, but in this case we lose compatibility with previous implementations of serialization. Is it possible?